couscous.yml

```diff
@@ -0,0 +1,38 @@
+template:
+    url: https://github.com/CouscousPHP/Template-Dark
+
+exclude:
+    - library
+    - vendor
+    - bin
+    - tests
+
+baseUrl: http://respect.github.io/Validation
+
+github:
+    user: Respect
+    repo: Validation
+
+title: Respect\Validation
+subTitle: The most awesome validation engine ever created for PHP
+
+menu:
+    sections:
+        main:
+            name: Table of contents
+            items:
+                home:
+                    text: Home
+                    relativeUrl: /
+                contributing:
+                    text: Contributing
+                    relativeUrl: CONTRIBUTING.html
+                documentation:
```

Comment:
How about put all rules in the menu? It will create some duplication but would be cool.